core_backend/app/routers/question_answer.py

```diff
@@ -38,45 +46,50 @@ async def llm_response(
 
     feedback_secret_key = generate_secret_key()
 
-    # add to query database
     user_query_db = await save_user_query_to_db(
         asession, feedback_secret_key, user_query
     )
 
-    # get FAQs from vector db
-    content_response = get_similar_content(
-        user_query, qdrant_client, int(QDRANT_N_TOP_SIMILAR)
-    )
-
-    # add FAQs to responses database
-    prompt = (
-        f"Answer the user query in square brackets:"
-        f"\n[{user_query.query_text}]\n"
-        f"in natural language by rewording the following FAQ found below. "
-        f"Address the question directly and do not respond with anything that "
-        f"is outwith the context of the given FAQ."
-        f"\nIf the FAQ doesn't seem to answer the question, respond with "
-        f"'Sorry, no relevant information found.'"
-        f"\n\nFound FAQ:\n{content_response[0].response_text}"
+    user_query_refined = UserQueryRefined(
+        **user_query.model_dump(), query_text_original=user_query.query_text
     )
 
-    # generate llm response
-    messages = [{"content": prompt, "role": "user"}]
-    llm_response_raw = completion(
-        model=OPENAI_LLM_TYPE, messages=messages, temperature=0
-    )
-    llm_text_response = llm_response_raw.choices[0].message.content
-
-    # format to response schema
     response = UserQueryResponse(
         query_id=user_query_db.query_id,
-        content_response=content_response,
-        llm_response=llm_text_response,
+        content_response=None,
+        llm_response="Sorry, we cannot do that.",
         feedback_secret_key=feedback_secret_key,
     )
+
+    # -- pre-process input
```

Comment:
Will refactor in future PR to take each of these out to a separate function. Make it a little easier to understand